pynuki/lock.py

```diff
@@ -10,6 +10,22 @@ class NukiLock(NukiDevice):
     def is_locked(self):
         return self.state == const.STATE_LOCK_LOCKED
 
+    @property
+    def is_door_sensor_activated(self):
+        return (
+            self.door_sensor_state != const.STATE_DOORSENSOR_DEACTIVATED
+            or self.door_sensor_state != const.STATE_DOORSENSOR_UNKNOWN
+            or self.door_sensor_state != None
+        )
```

Comment:
I've tried your PR with my v1 lock.

In its current state `is_door_sensor_activated` returns `True` for my lock.
I suggest to update the method as follows:

```suggestion
    @property                                                                                  
    def is_door_sensor_activated(self):
        # Nuki v1 locks don't have a door sensor, therefore the                                
        # door_sensor_state will is unset for them.                    
        if (
            not self.door_sensor_state
            or self.door_sensor_state == const.STATE_DOORSENSOR_UNKNOWN    
        ):                                     
            return
        return self.door_sensor_state != const.STATE_DOORSENSOR_DEACTIVATED
```

Reply:
Perfect, I added your suggestion. When you're ready I would stash my commits and we can merge :)

Reply:
I squashed the PR. Thanks for your contribution :)